docs/guides/extensions/panels-create-items.md

```diff
@@ -0,0 +1,566 @@
+---
+description: Learn how to create custom panel extensions with dynamic forms that enter items into a collection.
+contributors: Tim Butterfield, Kevin Lewis
+---
+
+# Create An Interactive Panel To Create Items
+
+Panels are used in dashboards as part of the Insights module. As well as read-only data panels, they can be interactive
+with form inputs. In this guide, you will create a panel that automatically generates a form based on a collection's
+fields, and allows item creation from an Insights dashboard.
+
+![A panel shows a form called Add Customer. It has a name, surname, and phone number text input.](https://marketing.directus.app/assets/5385ebc9-e3d9-4f75-9ec8-86bd54589d56.png)
+
+## Install Dependencies
+
+Open a console to your preferred working directory and initialize a new extension, which will create the boilerplate
+code for your operation.
+
+```
+npx create-directus-extension
```

Comment:
NPM has another alias for this: `npm init directus-extension`. I don't really care which one we use, as the result is the same, but lets make sure we're consistent in all these articles 🙂

Reply:
We are - we use npx